### ISSUE TITLE
Reset needed every screen wakeup

### ISSUE THREAD

**Issue by Dubardo** [valinet/ExplorerPatcher#2936]
I didn't find this issue but every time my screen turn off, I need to reset File explorer to make it work properly.

I have a 3 screen setup, 2x 1200p and 1x ultrawide 1440p.
My bar is on the left screen, right vertically and when I wakeup the screen, if I fullscreen a window, it will go over the bar.
The start menu will popoff the middle screen as would the default placement.
Also my ultrawide middle screen seems to be cut in half about 1/10 at the top. 

I will try to provide video next time it happens.

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.  These should be screen shots.

Please include any other software you have installed that modifies the UI of windows.

**Comment by Dubardo** [comment]
W11 Version 23H2 build 22631.3227
Explorer patch 22621.3007.63.2
![image](https://github.com/valinet/ExplorerPatcher/assets/18554856/9d467900-4652-44f2-a3f1-01bed24678a4)
![image](https://github.com/valinet/ExplorerPatcher/assets/18554856/8b2e1961-e5a1-4de1-b6dd-076fad890822)
![image](https://github.com/valinet/ExplorerPatcher/assets/18554856/283e46ef-b8b0-4186-9a3b-ca4549531183)
![image](https://github.com/valinet/ExplorerPatcher/assets/18554856/c479bf2d-b8f0-454a-96d3-44ad67e79eac)
![image](https://github.com/valinet/ExplorerPatcher/assets/18554856/4be999d1-a410-4732-865d-3b36a35202f7)
![image](https://github.com/valinet/ExplorerPatcher/assets/18554856/6d3020cd-3467-4f58-ba0c-41e6c1be6769)

I'm running PowerToys v0.43.0 for the fancy zone and Display Fusion Pro 10.1.2

**Comment by pyrates999** [comment]
Try uninstalling the preview update you got installed: https://answers.microsoft.com/en-us/insider/forum/all/build-226213227-and-226313227-live-in-rpc/7eb26fb6-64f8-491f-8599-4e969655f800

**Comment by Dubardo** [comment]
Killing Powertoys seems to have fixed it. I only use it for Fancy Zone so that might be it.

**Comment by pyrates999** [comment]
So if uninstalling powertoys fixes it, please mark it as the answer.

**Comment by Dubardo** [comment]
I re-installed the Microsoft store version of Powertoys and it seems to be ok now!

**Comment by pyrates999** [comment]
So it sounds like you just needed to update powertoys.

**Comment by Dubardo** [comment]
Nope, it didn't work. Still got the issue with updated version. Guess ill have to learn without the fancy zones!

**Comment by Dubardo** [comment]
It still does it without Fancytoys and Display fusion:

https://streamable.com/6xk1nq

And

![NVIDIA_GeForce_Overlay_2024-03-17_14-04-59](https://github.com/valinet/ExplorerPatcher/assets/18554856/8d3a2edc-d43f-440f-83d8-96615e9f8968)

**Comment by pyrates999** [comment]
For the reset screen problem, uninstall EP, reboot, and see if it is still happening.

**Comment by Simsala91** [comment]
I'm also having this issue whenever I change display settings. No Powertoys or Fancytoys installed. Fullscreen applications (like Firefox) will be displayed over the taskbar. Need to restart explorer everytime to fix.
